src/scss/base.scss

```diff
@@ -0,0 +1,286 @@
+html {
+    font-size: 14px;
+}
+
+body {
+    min-width: 1010px;
+
+    &.layout_frontend {
+        @media (prefers-color-scheme: dark) {
```

Comment:
Do you need these 2 `@media` blocks? The `themify` mixin is intended to handle this. Also these will only read the system colour theme, not the selected colour theme.

Reply:
This part is the initial loading screen. In the previous PR it was requested by somebody to add it in as if you're in dark theme you'd get a flash of white when loading the vault

Reply:
Resolved, see `index.html` and `misc.scss`